Sources/SwiftCliCore/StitchLibrary.swift

```diff
@@ -24,6 +24,6 @@ let allowedUserInput = allowedStitchesInfo.map { $0.name } + nonStitchAllowedInp
 func stitchLookup(stitch: String) throws -> StitchInfo {
     if let lookupStitch = allowedStitchesInfo.first(where: { $0.name == stitch }) {
         return lookupStitch
-    }
+        }
     throw StitchParsingError.invalidInput(stitch)
     }
```

Comment:
Indentation fix should be for this bottom closing brace. The previous closing brace was fine as it was!